app/helpers/notifications_helper.rb

```diff
@@ -23,4 +23,12 @@ def any_active_filters?
   def filtered_params(override = {})
     filters.merge(override)
   end
+
+  def archive_selected_button(custom_class=nil)
+    action = params[:archive] ? 'unarchive' : 'archive'
+    button_tag(type: "button",
+               class: "archive_toggle #{action}_selected #{custom_class}") do
+      "#{action} selected".capitalize
```

Comment:
Would it make sense to remove the `.capitalize` and replace 
`action = params[:archive] ? 'unarchive' : 'archive'`
with 
`action = params[:archive] ? 'Unarchive' : 'Archive'`?
I think it would be the same, but correct me if I'm wrong.